crates/weaver_codegen_test/templates/registry/rust/attributes/mod.rs.j2

```diff
@@ -10,11 +9,12 @@
 
 use opentelemetry::{Key, KeyValue, StringValue};
 
+/// A constant indicating whether the attributes are enabled.
 pub const ATTRIBUTES_PARAM: bool = {{ params.attributes }};
 
 {% for attributes_root_namespace in ctx %}
-/// Attributes for the `{{ attributes_root_namespace.root_namespace }}` namespace.
-pub mod {{ attributes_root_namespace.root_namespace | snake_case }};
+//! Attributes for the `{{ attributes_root_namespace.root_namespace }}` namespace.
+//! pub mod {{ attributes_root_namespace.root_namespace | snake_case }};
```

Comment:
Sorry, I didn’t understand these changes. Why comment out this part of the code generation?

Reply:
Clippy was complaining about the documentation and formatting. The jinja template previously created:
```rust
/// Attributes for the `system` namespace.
pub mod system;
```
for example.

Then here: https://github.com/open-telemetry/weaver/blob/42203247a8217ef6909949764ab641956502e8ff/crates/weaver_codegen_test/build.rs#L84-L89 the `pub mod` lines are removed leaving a floating `///` doc comment.

Reply:
Ah, ok. I had forgotten about that part of the plumbing in the code :-)